.github/workflows/slack.yml

```diff
@@ -13,4 +13,4 @@ jobs:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
       uses: pullreminders/slack-action@master
       with:
-        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred the Helm Chart Releaser! (https://github.com/layer5io/helm-chart-releaser/stargazers) Total ⭐️: ${{env.STARS}}\"}'
+        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"${{ github.actor }} just starred the Helm Chart Releaser! (https://github.com/layer5io/helm-chart-releaser/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```

Comment:
Can we make all of these slack.yml workflows more generic by using a built-in variable for the repo name?￼